src/main/java/domain/WinningLottoTicket.java

```diff
@@ -0,0 +1,23 @@
+package domain;
+
+public class WinningLottoTicket {
+
+	private final LottoTicket winningLottoTicket;
+	private final LottoNumber bonusNumber;
+
+	public WinningLottoTicket(LottoTicket winningLottoTicket, LottoNumber bonusNumber) {
+		validateDuplicateBonusNumber(winningLottoTicket, bonusNumber);
+		this.winningLottoTicket = winningLottoTicket;
+		this.bonusNumber = bonusNumber;
+	}
+
+	private void validateDuplicateBonusNumber(LottoTicket winningLottoTicket, LottoNumber bonusNumber) {
+		if (winningLottoTicket.contains(bonusNumber)) {
+			throw new IllegalArgumentException(String.format("보너스 번호는 당첨 번호에 포함되선 안됩니다. LottoTicket: %s, bonusNumber: %s", winningLottoTicket, bonusNumber));
+		}
+	}
+
+	public LottoWinningRank calculateResult(LottoTicket lottoTicket) {
+		return LottoWinningRank.calculateRank(lottoTicket.countSameNumbers(winningLottoTicket), lottoTicket.contains(bonusNumber));
```

Comment:
코드의 가독성을 위해서 메서드 파라미터로 곧바로 전달하는 로직들을 적절히 변수로 뽑아주면 좋을 것 같아요. 😉
```suggestion
		var matchCount = lottoTicket.countSameNumbers(winningLottoTicket);
		var hasBonusNumber = lottoTicket.contains(bonusNumber);
		
		return LottoWinningRank.calculateRank(matchCount, hasBonusNumber);
```